src/react-app-env.d.ts

```diff
@@ -1 +1,35 @@
 /// <reference types="react-scripts" />
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {
+    street: string;
+    suite: string;
+    city: string;
+    zipcode: string;
+    geo: {
+      lat: string;
+      lng: string;
+    };
+  };
+  phone: string;
+  website: string;
+  company: {
+    name: string;
+    catchPhrase: string;
+    bs: string;
+  };
```

Comment:
for this particular task it's fine, 
but imagine that you'd need to work with Company as a standalone instance - you'd need a type / interface for it
so advise is to introduce interfaces for any nested types:
```suggestion
  company: Company;
  
 interface Company {
    name: string;
    catchPhrase: string;
    bs: string;
  };
```